src/modules/events/send_event.h

```diff
@@ -51,46 +51,92 @@ extern "C" __EXPORT int send_event_main(int argc, char *argv[]);
 class SendEvent : public ModuleBase<SendEvent>, public ModuleParams
 {
 public:
+
+	/**
+	 * @brief Default Constructor.
+	 */
 	SendEvent();
-	~SendEvent();
 
 	/**
-	 * Initialize class in the same context as the work queue. And start the background listener.
-	 * @return 0 if successful, <0 on error */
-	static int task_spawn(int argc, char *argv[]);
+	 * @brief Default Destructor.
+	 */
+	~SendEvent();
 
-	/** @see ModuleBase */
+	/**
+	 * @brief
+	 * @param argc The task argument count.
+	 * @param argc Pointer to the task argument variable array.
+	 * @see ModuleBase.
```

Comment:
Does having the full stop render OK?

Reply:
I don't know... That was actually there from before.  Would you advise?